### PR TITLE
packages/contracts: Remove legacy `bin/deploy.ts` script

### PR DESCRIPTION
**Pull request by mslipper**
Removes the `bin/deploy.ts` script, which was a holdover from before the regenesis and was little more than a wrapper around `npx hardhat deploy`. The functionality that used to live in `bin/deploy.ts` has been moved into `ops/scripts/deployer.sh`, since the old script's functionality was exclusively run in CI.

Metadata:

- Fixes ENG-1447

### CODE REVIEW

File: ops/docker/Dockerfile.deployer [hunk 1]
```diff
@@ -34,4 +34,4 @@ COPY packages/contracts/test/helpers/constants.ts ./test/helpers/constants.ts
 COPY packages/contracts/scripts ./scripts
 
 COPY ./ops/scripts/deployer.sh .
-ENTRYPOINT yarn run deploy
+CMD ./ops/scripts/deployer.sh
```
Comment (tynes) [inline ops/docker/Dockerfile.deployer:37]: It should be fine to skip using `yarn` for the deployer, but other services may consume env vars injected into the runtime from `yarn`

File: ops/scripts/deployer.sh [hunk 2]
```diff
@@ -14,30 +19,82 @@ curl \
...
+concat_arg "--forked" "FORKED"
+
+echo "Deploying contracts. Deployment command:"
+echo "$DEPLOY_CMD"
+eval "$DEPLOY_CMD"
+
+echo "Building addresses.json."
+export ADDRESS_MANAGER_ADDRESS=$(cat "./deployments/$CONTRACTS_TARGET_NETWORK/Lib_AddressManager.json" | jq -r .address)
+
+# First, create two files. One of them contains a list of addresses, the other contains a list of contract names.
```
Comment (tynes) [inline ops/scripts/deployer.sh:68]: Nice bash wizardry

**Review by tynes — APPROVED** [review]

**Review by smartcontracts — APPROVED** [review]
Nice, been wanting this removed for a while.

**Review by tynes — CHANGES_REQUESTED** [review]
Just realized this should have a changeset for the contracts so it versions the deployer docker image

**Review by tynes — APPROVED** [review]

**Pull request merged**